### PR TITLE
(COURSE ):Free Courses of language kannada added

### PR DESCRIPTION
**Pull request by Atharva1723**
## What does this PR do?
I have added some links to free course for various programming languages

### Why is this valuable (or not)?
This is valuable as the md file for kannada language was not added before
### How do we know it's really free?
You can test the links

### For book lists, is it a book? For course lists, is it a course? etc.
I have attached links in the readme file for free courses  
## Checklist:
- [✓ ] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [✓ ] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [ ✓] Include author(s) and platform where appropriate.
- [✓ ] Put lists in alphabetical order, correct spacing.
- [ ✓] Add needed indications (PDF, access notes, under construction).
- [✓ ] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Comment by sohansai** [comment]
Check the status of GitHub Actions and resolve reported warnings.

**Comment by Atharva1723** [comment]
I have changed accordingly

**Comment by Atharva1723** [comment]
Please merge this pull request

**Comment by sohansai** [comment]
Before we proceed with merging this pull request please make sure to review the status of GitHub Actions and resolve reported warnings that have been reported.

Make sure to follow the CONTRIBUTING.md guidelines before contributing.

**Comment by eshellman** [comment]
see #9072 
Need to add link in README.md
language code for kannada is `kn`, not `ka` - change the file name.
Be sure to check the free-programming-books-lint output for linter errors. Alphabetical order and spacing are the most common problems.
We are not labeling our course linsts with difficulty level.
For youtube, we prefer "playlist" links and the name of the producer/creator. docs/CONTRIBUTING.md#platforms-and-access-notes.
We do not allow shortened links.

**Comment by Atharva1723** [comment]
@eshellman I have made necessary changes in the Pull request.Please add me as assignee and add hacktoberfest and hacktoberfest-accepted label and please merge this pr
Thank you.

File: courses/free-courses-kn.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+* [C](#c)
+* [HTML and CSS](#html-and-css)
+* [JavaScript](#javascript)
+* [PLC](#plc)
+* [Python](#python)
+
+
+### C
+
+* [Complete C Language Course in KANNADA](https://youtu.be/OvyNtFgn8T0?si=Bb0rLk3Cfl0zMcaU) - The Coding Class: Yuvaraj Madha
```
Comment (eshellman) [inline courses/free-courses-kn.md:12]: This is a shortened link

File: courses/free-courses-kn.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+* [C programming in Kannada](https://youtu.be/MhRzCX18mDE?si=vp89xBkxJxB1PdfZ) - script kiddie
```
Comment (eshellman) [inline courses/free-courses-kn.md:13]: This is a shortened link

File: courses/free-courses-kn.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+
+### HTML and CSS
+
+* [CSS in Kannada (ಕನ್ನಡದಲ್ಲಿ) - Beginner to Advanced.](https://youtu.be/wAqQScpoOMQ?si=Lx8zLwW1lm-sqBeL) - DEBUG CODING
```
Comment (eshellman) [inline courses/free-courses-kn.md:18]: This is a shortened link

File: courses/free-courses-kn.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+
+### PLC
+
+* [PLC tutorial for beginners](https://youtube.com/playlist?list=PLM-fDuwhsV0nAyn-B06TTbDW78HL3pNiw&si=zaboqVNmCvXMdk6F) - DevelUp Technical
+
+
+### Python
+
+* [Learn Python in Kannada](https://youtu.be/MhktLoePTiY?si=bEP4D8NEVoBdb0GX) - Engineering in Kannada
```
Comment (eshellman) [inline courses/free-courses-kn.md:35]: This is a shortened link

File: courses/free-courses-kn.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+* [python complete course in Kannada](https://youtube.com/playlist?list=PLUZkVL-W-8GKpo--HuELu27Lkc308fNXe&si=NS4YhygcUNHqtTwm) - script kiddie
```
Comment (eshellman) [inline courses/free-courses-kn.md:36]: remove the tracking parameter.
```suggestion
* [python complete course in Kannada](https://youtube.com/playlist?list=PLUZkVL-W-8GKpo--HuELu27Lkc308fNXe) - script kiddie
```

File: courses/free-courses-kn.md [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+
+### JavaScript
+
+* [Javascript tutorial in Kannada || Javascript in Kannada || Complete Javascript in Kannada](https://youtube.com/playlist?list=PLBVoFEaXQzK0ybl-O3g_2_PswaLzNGbb8&si=EwrfyaDqm6G3MJnb) - Learn Tech in Kannada
+* [JavaScript course in Kannada](https://youtube.com/playlist?list=PLUZkVL-W-8GJVkp8Az0SAWqmDPv5b2Tn9&si=BsVZ8lpgZi024CXr) - script kiddie
+
+
+### PLC
+
+* [PLC tutorial for beginners](https://youtube.com/playlist?list=PLM-fDuwhsV0nAyn-B06TTbDW78HL3pNiw&si=zaboqVNmCvXMdk6F) - DevelUp Technical
```
Comment (eshellman) [inline courses/free-courses-kn.md:30]: remove the tracking parameter.

File: courses/free-courses-kn.md [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+
+### HTML and CSS
+
+* [CSS in Kannada (ಕನ್ನಡದಲ್ಲಿ) - Beginner to Advanced.](https://youtu.be/wAqQScpoOMQ?si=Lx8zLwW1lm-sqBeL) - DEBUG CODING
+* [HTML and CSS in Kannada || Web Designing in Kannada || CSS in Kannada || Html in Kannada](https://youtube.com/playlist?list=PLBVoFEaXQzK3F2BN7ZbXb_EpnDMAlJsPo&si=xrva9ZdT93aR4n2N) - Learn Tech in Kannada
+
+
+### JavaScript
+
+* [Javascript tutorial in Kannada || Javascript in Kannada || Complete Javascript in Kannada](https://youtube.com/playlist?list=PLBVoFEaXQzK0ybl-O3g_2_PswaLzNGbb8&si=EwrfyaDqm6G3MJnb) - Learn Tech in Kannada
```
Comment (eshellman) [inline courses/free-courses-kn.md:24]: remove the tracking parameter.

File: courses/free-courses-kn.md [hunk 8]
```diff
@@ -0,0 +1,36 @@
...
+* [JavaScript course in Kannada](https://youtube.com/playlist?list=PLUZkVL-W-8GJVkp8Az0SAWqmDPv5b2Tn9&si=BsVZ8lpgZi024CXr) - script kiddie
```
Comment (eshellman) [inline courses/free-courses-kn.md:25]: remove the tracking parameter.

File: courses/free-courses-kn.md [hunk 9]
```diff
@@ -0,0 +1,36 @@
...
+### C
+
+* [Complete C Language Course in KANNADA](https://youtu.be/OvyNtFgn8T0?si=Bb0rLk3Cfl0zMcaU) - The Coding Class: Yuvaraj Madha
+* [C programming in Kannada](https://youtu.be/MhRzCX18mDE?si=vp89xBkxJxB1PdfZ) - script kiddie
+
+
+### HTML and CSS
+
+* [CSS in Kannada (ಕನ್ನಡದಲ್ಲಿ) - Beginner to Advanced.](https://youtu.be/wAqQScpoOMQ?si=Lx8zLwW1lm-sqBeL) - DEBUG CODING
+* [HTML and CSS in Kannada || Web Designing in Kannada || CSS in Kannada || Html in Kannada](https://youtube.com/playlist?list=PLBVoFEaXQzK3F2BN7ZbXb_EpnDMAlJsPo&si=xrva9ZdT93aR4n2N) - Learn Tech in Kannada
```
Comment (eshellman) [inline courses/free-courses-kn.md:19]: remove the tracking parameter.

**Review by eshellman — CHANGES_REQUESTED** [review]
getting there! Please remove tracking parameters and expand shortened links

File: courses/free-courses-kn.md (hunk 9 above)
Comment (Atharva1723) [reply courses/free-courses-kn.md]: @eshellman I have made necessary changes in the Pull request.Please add me as assignee and add hacktoberfest and hacktoberfest-accepted label and please merge this pr
Thank you.

**Comment by Atharva1723** [comment]
@eshellman I have made necessary changes in the Pull request.Please add me as assignee and add hacktoberfest and hacktoberfest-accepted label and please merge this pr
Thank you.

File: courses/free-courses-kn.md [hunk 10]
```diff
@@ -0,0 +1,37 @@
...
+* [HTML and CSS](#html-and-css)
+* [JavaScript](#javascript)
+* [PLC](#plc)
+* [Python](#python)
+
+
+### C
+
+* [Complete C Language Course in KANNADA](https://youtube.com/playlist?list=PLjvPj4x59YBUYkNJr4kCKOXsP3NyM4-ik) - The Coding Class: Yuvaraj Madha
+* [C programming in Kannada](https://youtube.com/playlist?list=PLUZkVL-W-8GLVwteCNH_HNoIAhbfBHnLb) - script kiddie
```
Comment (eshellman) [inline courses/free-courses-kn.md:13]: `C p` should come before `Com` alphabetically

File: courses/free-courses-kn.md [hunk 11]
```diff
@@ -0,0 +1,37 @@
...
+
+
+### HTML and CSS
+
+* [HTML in Kannada (ಕನ್ನಡದಲ್ಲಿ) - Beginners to Advanced](https://youtube.com/playlist?list=PLBGSzVCM24iHCjyPxCZBbZSNIiFS7vDFl) - DEBUG CODING
+* [CSS in Kannada (ಕನ್ನಡದಲ್ಲಿ) - Beginner to Advanced.](https://youtube.com/playlist?list=PLBGSzVCM24iHnpfOMnhuyiyEo_NGSxdPD) - DEBUG CODING
```
Comment (eshellman) [inline courses/free-courses-kn.md:19]: comes first aphabetiacally

File: courses/free-courses-kn.md [hunk 12]
```diff
@@ -0,0 +1,37 @@
...
+* [HTML and CSS in Kannada || Web Designing in Kannada || CSS in Kannada || Html in Kannada](https://youtube.com/playlist?list=PLBVoFEaXQzK3F2BN7ZbXb_EpnDMAlJsPo) - Learn Tech in Kannada
+
+
+### JavaScript
+
+* [Javascript tutorial in Kannada || Javascript in Kannada || Complete Javascript in Kannada](https://youtube.com/playlist?list=PLBVoFEaXQzK0ybl-O3g_2_PswaLzNGbb8) - Learn Tech in Kannada
+* [JavaScript course in Kannada](https://youtube.com/playlist?list=PLUZkVL-W-8GJVkp8Az0SAWqmDPv5b2Tn9) - script kiddie
```
Comment (eshellman) [inline courses/free-courses-kn.md:26]: 1st alphabetically

File: courses/free-courses-kn.md [hunk 13]
```diff
@@ -0,0 +1,37 @@
...
+
+### C
+
+* [Complete C Language Course in KANNADA](https://youtube.com/playlist?list=PLjvPj4x59YBUYkNJr4kCKOXsP3NyM4-ik) - The Coding Class: Yuvaraj Madha
+* [C programming in Kannada](https://youtube.com/playlist?list=PLUZkVL-W-8GLVwteCNH_HNoIAhbfBHnLb) - script kiddie
+
+
+### HTML and CSS
+
+* [HTML in Kannada (ಕನ್ನಡದಲ್ಲಿ) - Beginners to Advanced](https://youtube.com/playlist?list=PLBGSzVCM24iHCjyPxCZBbZSNIiFS7vDFl) - DEBUG CODING
```
Comment (eshellman) [inline courses/free-courses-kn.md:18]: last alphabetically

**Review by eshellman — CHANGES_REQUESTED** [review]
Sorry, our linter is not checking Kannada alphabetical order. I think this is the last thing to fix.

**Comment by Atharva1723** [comment]
@eshellman Also made the alphabetical order changes

**Comment by eshellman** [comment]
Fantastic work! 
Thanks, and welcome to free-programming-books!

**Comment by Atharva1723** [comment]
Thank you for merging this pr ...will work more on this repo

**Pull request merged**